Assets/Scripts/Plant.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Burst.CompilerServices;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+[CreateAssetMenu(menuName = "Plants/base plant", fileName = "NewPlantData")]
+public class Plant : ScriptableObject
+{
+    //Potencjalne problemy do omowienia:
+    //>Co z grafikami? Kopiujac zajmujemy wiecej pamieci? Moze klasa PlantData
+    //przechowujaca je w jednym miejscu caly czas? Ale jak ja wtedy zapisac? ID? NameID?
+    //>Stan rosliny powinien byc w SO? Czy nie bedziemy tego naduzywac i wystarczy enum?
```

Comment:
Nie potrafię sobie tego wyobrazić w SO więc wydaje mi się że enum wystarczy

Reply:
W takim razie musimy pamiętać by nie komplikować zbyt wielu stanów.
Na razie są stany "growing" i "ready to harvest".  Potem możemy dodać coś w stylu "dead" albo "cant grow" kiedy pora roku się nie zgadza albo coś